src/modules/fancyzones/editor/FancyZonesEditor/CanvasEditorWindow.xaml

```diff
@@ -49,7 +49,8 @@
                     ToolTip="{x:Static props:Resources.Add_zone}"
                     DataContext="{Binding Path=Model, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=Window}}"
                     IsEnabled="{Binding IsZoneAddingAllowed}"
-                    Click="OnAddZone" />
+                    Click="OnAddZone"
+                    Value="{x:Static props:Resources.New_zone_added}" />
```

Comment:
I think `Value` should have a more meaningful name, e.g. `NarratorName` or something, since just Value could be misleading.

Reply:
Renamed to OnClickAutomationValue